src/tests/functional/shared_test_classes/src/single_op/activation.cpp

```diff
@@ -8,10 +8,160 @@
 #include "openvino/op/parameter.hpp"
 #include "openvino/op/constant.hpp"
 #include "openvino/op/result.hpp"
+#include <common_test_utils/ov_tensor_utils.hpp>
 
 namespace ov {
 namespace test {
 using ov::test::utils::ActivationTypes;
+void ActivationLayerTest::generate_inputs(const std::vector<ov::Shape>& targetInputStaticShapes) {
```

Comment:
Why required? May we modify `src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp` per specific layer?

Reply:
It's needed for fp16 precision.